scalastyle-config.xml

```diff
@@ -250,6 +250,14 @@ This file is divided into 3 sections:
     <customMessage>Omit braces in case clauses.</customMessage>
   </check>
 
+  <!-- SPARK-16877: Avoid Java annotations -->
+  <check customId="OverrideJavaCase" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
+    <parameters><parameter name="regex">^Override$</parameter></parameters>
```

Comment:
Yes, I just ran a test after cloning [scalariform](https://github.com/scala-ide/scalariform) which [scalastyle](https://github.com/scalastyle/scalastyle) uses as below:

```
ScalaLexer.rawTokenise("@Override")
```

It seems this becomes

![2016-08-04 1 29 11](https://cloud.githubusercontent.com/assets/6477701/17390365/78afcb96-5a47-11e6-9e9f-8d9a2d6c4ddf.png)

different tokens.

(BTW, maybe we should avoid to write `@Override` as it is.. I started to feel guilty for cc him/her here and there....)